### PR TITLE
dr-auto-sync: enable min-resolved-ts persist by default

### PR DESCRIPTION
**Pull request by lhy1024**
Signed-off-by: lhy1024 <admin@liudos.us>

### What problem does this PR solve?

Issue Number: Ref https://github.com/tikv/pd/issues/4399

### Check List

Tests

- Unit test
- Integration test

### Release note

```release-note
dr-auto-sync: enable min-resolved-ts persist by default
```

### CODE REVIEW

File: server/cluster/cluster.go [hunk 1]
```diff
@@ -62,7 +62,8 @@ import (
 
 var (
 	// DefaultMinResolvedTSPersistenceInterval is the default value of min resolved ts persistence interval.
-	DefaultMinResolvedTSPersistenceInterval = 10 * time.Second
+	// If interval in config is zero, it means not to persist resolved ts and check config with this DefaultMinResolvedTSPersistenceInterval
+	DefaultMinResolvedTSPersistenceInterval = time.Second
```
Comment (nolouch) [inline server/cluster/cluster.go:66]: Why need to reduce the interval?

Comment (rleungx) [inline server/cluster/cluster.go:66]: Why not keep it as before?

Comment (lhy1024) [reply server/cluster/cluster.go:66]: This default interval is also the interval for detecting whether the config has changed. During the manual test, it was found that the previous configuration was too long, and it took ten seconds for the configuration to take effect.

Comment (lhy1024) [reply server/cluster/cluster.go:66]: This default interval is also the interval for detecting whether the config has changed. During the manual test, it was found that the previous configuration was too long, and it took ten seconds for the configuration to take effect.

File: server/config/config.go [hunk 2]
```diff
@@ -236,7 +236,7 @@ const (
 	defaultFlowRoundByDigit                 = 3 // KB
 	maxTraceFlowRoundByDigit                = 5 // 0.1 MB
 	defaultMaxResetTSGap                    = 24 * time.Hour
-	defaultMinResolvedTSPersistenceInterval = 0
+	defaultMinResolvedTSPersistenceInterval = time.Second
```
Comment (nolouch) [inline server/config/config.go:239]: Can it shared with `DefaultMinResolvedTSPersistenceInterval`?

**Comment by rleungx** [comment]
CI failed

**Review by disksing — APPROVED** [review]

**Comment by nolouch** [comment]
ci failed.

**Comment by lhy1024** [comment]
> ci failed.

I fixing it.

**Review by nolouch — APPROVED** [review]
lgtm

**Pull request merged**